### ISSUE TITLE
UNABLE TO FILE - MULTIPLE PASSENGER TYPES

### ISSUE THREAD

**Issue by felradces** [Travelport/travelport-uapi-tutorial-c-sharp#70]
Hi , 

I don't know what's missing on my xml, can you pinpoint, what's missing? or what caused the response message in AirCreateReservationResp:

<ResponseMessage Code="0" Type="Warning" ProviderCode="1G" xmlns="http://www.travelport.com/schema/common_v39_0">UNABLE TO FILE - MULTIPLE PASSENGER TYPES</ResponseMessage>
  <ResponseMessage Code="0" Type="Warning" ProviderCode="1G" xmlns="http://www.travelport.com/schema/common_v39_0">Some of the requested AirPricingInfos could not be saved for the requested provider.</ResponseMessage>

I do have AirPricingInfo for both ADULT and CHILD in AirCreateReservationReq, 
I used the AirPricingInfos given by AirPriceResponse.

I hope you can give me a clear answer.
Let me know.

See my attached file for reference:
[LowFareSearchReq.txt](https://github.com/Travelport/travelport-uapi-tutorial-c-sharp/files/905168/LowFareSearchReq.txt)
[LowFareSearchResp.txt](https://github.com/Travelport/travelport-uapi-tutorial-c-sharp/files/905169/LowFareSearchResp.txt)
[AirPriceReq.txt](https://github.com/Travelport/travelport-uapi-tutorial-c-sharp/files/905170/AirPriceReq.txt)
[AirPriceResp.txt](https://github.com/Travelport/travelport-uapi-tutorial-c-sharp/files/905171/AirPriceResp.txt)
[AirCreateReservationReq.txt](https://github.com/Travelport/travelport-uapi-tutorial-c-sharp/files/905172/AirCreateReservationReq.txt)
[AirCreateReservationResp.txt](https://github.com/Travelport/travelport-uapi-tutorial-c-sharp/files/905173/AirCreateReservationResp.txt)

Regards

**Comment by vivekjyotipramanik** [comment]
Hi felradces,

I am not seeing any issues with the Request. Could you please try with a different carrier like QF and let me know if you are getting the same issues with QF. Thanks.

**Comment by felradces** [comment]
actually, this is already a pre-prod testing, almost all of the Carrier choice are PR only, As you can see on the LowFareSearchResponse.

**Comment by vivekjyotipramanik** [comment]
Hi felradces,

If this is preProduction, Please raise an ORS ticket and an API Support Analyst will assist you regarding this. Thanks.

**Comment by felradces** [comment]
yep, i already raised this, but i'm clueless right now what to fix or missing on this sample request.

**Comment by vivekjyotipramanik** [comment]
Hi felradces,

Thanks for raising the ORS ticket, an API analyst will check the issue and respond you as soon as possible. 
I could not find any issues with the request, so it might be a carrier settings or pricing issue. An analyst will check this in depth and will respond you through ORS. Thanks.
